migrations/20230125174531_backfill_invite_identities.up.sql

```diff
@@ -0,0 +1,6 @@
+-- backfills the missing email identity for invited users
+
+insert into auth.identities (id, user_id, identity_data, provider, last_sign_in_at, created_at, updated_at)
+select id, id as user_id, jsonb_build_object('sub', id, 'email', email) as identity_data, 'email' as provider, null as last_sign_in_at, '2023-01-25' as created_at, '2023-01-25' as updated_at
+from auth.users as users
+where invited_at is not null;
```

Comment:
Might be safer to check if there is no such identity, and fill it only for them? Otherwise the insert will fail?